feat!: Update llama guard and translation convenience function (#1007)
SAP/ai-sdk-js: V2-Upgrade-Guide.md

  violence: 'ALLOW_ALL'
})
```
#### Llama Guard Filter Changes

> The `buildLlamaGuardFilter()` function is now renamed to `buildLlamaGuard38BFilter()` function and now requires a `type` parameter as the first argument to distinguish between input and output filter configurations, and accepts categories as an array instead of individual parameters.

Reviewer: ```suggestion
The `buildLlamaGuardFilter()` function has been renamed to `buildLlamaGuard38BFilter()` function.
It now requires a `type` parameter as the first argument to distinguish between `input` and `output` filter configurations, and accepts categories as an array instead of individual parameters.
```

New line for each sentence.
